packages/rum-react/src/domain/performance/getTimer.ts

```diff
@@ -0,0 +1,23 @@
+export function getTimer() {
+  let duration: number
+  let startTime: number
+
+  function startTimer() {
+    const start = performance.now()
+    startTime = performance.timeOrigin + start
+  }
+
+  function stopTimer() {
+    duration = performance.timeOrigin + performance.now() - startTime
```

Comment:
**💭 thought:** ‏I was considering to suggest using functions from `timeUtils` (ex: `timeStampNow()` etc), similarly to the rest of the SDK including Duration Vitals. But then we would use `Date.now()` instead of `performance.now()`, which is somewhat less precise. So using `performance.now()` here *might* make sense, but then why not use it in Vitals?

🤔 🤔